src/Command/WorkoutCommand.php

```diff
@@ -75,7 +77,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $handlerOptions->setPassword($password);
         $handlerOptions->setPrefix($prefix);
         $handlerOptions->setDryrun($dryrun);
-        $handlerOptions->setDelete($delete);
+        $handlerOptions->setDelete($delete||$deleteOnly);
```

Comment:
Is this necessary since deleteOnly is only used?

Reply:
> Is this necessary since deleteOnly is only used?

Yes it is required for the command below to work as expected (delete workouts but do not create new ones):
```console
./bin/console garmin:workout draft-plan.csv import --delete-only -n
```
'$handlerOptions->getDelete()' has to return 'true' otherwise there deletion step will not be performed in 'deleteWorkouts' function
https://github.com/Raistlfiren/garmin-csv-plan/blob/537225ac4b3f89d704ae24544f99b9f8d1ea2e56/src/Library/Handler/AbstractHandler.php#L120

Or it would require to set both parameter 'delete' and 'delete-only' on the command line (not so obvious)
```console
./bin/console garmin:workout draft-plan.csv import --delete --delete-only -n
```

Reply:
Oh yeah, I gotcha. That makes since. I should probably add some documentation on these flags eventually since we are implementing more than the existing CSV importer.